java/src/test/java/ai/onnxruntime/OnnxTensorTest.java

```diff
@@ -88,6 +88,91 @@ public void testScalarCreation() throws OrtException {
     }
   }
 
+  @Test
+  public void testBufferCreation() throws OrtException {
+    OrtEnvironment env = OrtEnvironment.getEnvironment();
+
+    // Test creating a value from an array
+    // Arrays result in tensors allocated by ORT, so they do not have a backing java.nio.Buffer
```

Comment:
Perhaps, we can add documentation here and in the API if not there, where we copy data to native ORT buffers and where there is not a copy involved.

Reply:
This is described in the docs for the various `OnnxTensor.createTensor` methods.